docs/manual/docs/cmd/actions/deploy.md

```diff
@@ -0,0 +1,30 @@
+# deploy action
+
+This action adds, deploys and installs (only to site collection) an app.
```

Comment:
Could you please add `Github` so it becomes `Github action`?

Reply:
Sure. Will do when I update the PR.